virttest/data_dir.py

```diff
@@ -214,8 +216,22 @@ def get_tmp_dir(public=True):
 
 
 def get_download_dir():
-    if not os.path.isdir(DOWNLOAD_DIR):
-        shutil.copytree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
+    """
+    Returns the dir that holds download definition files, AKA asset files
+
+    Before retuning the download directory, it makes sure the user, writable
+    copy of the download dir is created, and that all asset definition
+    file are in sync.
+
+    In sync means that newer or non-existing asset definition files
+    are copied to the user's directory.  If the user's directory
+    contains newer files, they're not overwritten.  This sync behavior
+    is based on :func:`distutils.file_util.copy_file` parameter
+    "update".
+
+    :returns: a writable directory populated with asset definition files
+    """
+    dir_util.copy_tree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR, update=1)
```

Comment:
```
    If 'update' is true, 'src' will only be copied if 'dst' does not exist,
    or if 'dst' does exist but is older than 'src'.
```

That means if the user updates the file and then we update the file in git, it'll be overwritten. How about the `vt-bootstrap --vt-update-config` approach, wouldn't that be better?

Actually I forgot how `--vt-update-config` works, so instead I think it should be yet another (interactive) step in `vt-bootstrap` and if the `--vt-update-config` is used it should force-update. At least that's what I'd expect it to do as these are (asset) conifg files.